### PR TITLE
docs: Add README

### PR DESCRIPTION
**Pull request by bhelx**
Just copying over the doc from extism.org for now so we can delete that one https://extism.org/docs/integrate-into-your-codebase/php-host-sdk. We can update this after merge.

### CODE REVIEW

**Comment by nilslice** [comment]
@G4Vi, just testing some notification activity in Slack and using your handle randomly. Feel free to ignore these & sorry if you get bombarded with messages.

**Review by G4Vi — COMMENTED** [review]
Got the reminder. Let me know if I should actually review.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,57 @@
...
+> curl https://raw.githubusercontent.com/extism/extism/main/wasm/code.wasm > code.wasm
+> ```
+
+```php title=index.php
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+// See the manifest docs for more options https://extism.org/docs/concepts/manifest
+$wasm = (object) [ 'wasm' => [](object) [ 'path'] = './code.wasm']];
```
Comment (chrisdickinson) [inline README.md:34]: on php v8.3.0 (installed via homebrew) this throws a syntax error.
```suggestion
$wasm = (object) [ 'wasm' => (object) ['path' => './code.wasm']];
```

**Review by chrisdickinson — APPROVED** [review]

**Comment by bhelx** [comment]
This is no longer valid I think

**Pull request closed without merging**